MaxText/scratch_code/golden_llama3_1_export.py

```diff
@@ -69,8 +70,8 @@ def save_golden_logits(model_id, output_path):
   with jsonlines.open(output_path, "w") as f:
     f.write_all(all_data_to_save)
 
-  upload_blob("maxtext-llama", output_path, "Llama3_1_8B/golden-logits/" + output_path)
-  print("File {} uploaded to {}.".format(output_path, "Llama3_1_8B/golden-logits/" + output_path))
+  upload_blob("maxtext-llama", output_path, f"Llama3_1_8B/golden-logits/{output_path}")
```

Comment:
can we use an f string for the entire string here, also the next line

Reply:
Ok